core/src/main/scala/tamer/kafka/Kafka.scala

```diff
@@ -20,10 +18,10 @@ import zio.kafka.consumer.{CommittableRecord, Consumer, ConsumerSettings, Subscr
 import zio.kafka.producer.{Producer, ProducerSettings}
 import zio.stream.ZStream
 
-final case class StateKey(queryHash: String, groupId: String)
+final case class StateKey(stateKey: String, groupId: String)
 
 object Kafka {
-  trait Service {
+  sealed trait Service {
```

Comment:
actually if we do this we won't be able to create a test layer, assuming we would want one that mimics Kafka semantics using in memory data structures, unless it's right in this file (which I personally dislike)

Reply:
yes I considered that. My question then if we will ever write a test version of this
https://github.com/laserdisc-io/tamer/pull/212/files#diff-529150eccefe4191efb1d9bdf54d903d07cba34175f7baaa0bf1c24d221f7a5fR42-R72
that's probably the logic we will want to test most of the time. 
Anyway it's not a big deal, we can get rid of `sealed` and see how it evolves. It's really minor.